hash_chunker/hash_chunker.py

```diff
@@ -99,3 +104,28 @@ def _position_to_hex(self, position: int) -> str:
             hexed = self.hex_zero * zeros_count + hexed
         hexed += self.hex_zero * (self.hash_max_length - len(hexed))
         return hexed[: self.chunk_hash_length]
+
+    def _validate(
+        self,
+        chunk_size: int = 1,
+        all_items_count: int = 0,
+        chunks_count: int = 1,
+    ) -> None:
+        check_args_dict = {
+            "Chunk_size": chunk_size,
+            "All_items_count": all_items_count,
+            "Chunks_count": chunks_count,
+        }
+
+        for key, arg in check_args_dict.items():
+            if not isinstance(arg, int):
+                raise TypeError(f"{key} should be integer.")
+            if key == "All_items_count" and arg < 0:
+                raise ValueError(f"{key} should be great or equal 0.")
+        self._check_positive_values(check_args_dict)
+
+    def _check_positive_values(self, args: Dict) -> None:
```

Comment:
We can remove useless data from args outside the function to avoid check for key name in loop.